### PR TITLE
Coovi

### PR DESCRIPTION
**Pull request by CooviM**
file arrangement.

### CODE REVIEW

File: amigos/amigos/Weather_Sensor_Data.py [hunk 1]
```diff
@@ -15,7 +15,15 @@
 port = serial.Serial("/dev/ttyS5")
 port.baudrate = 115200
 t=0
+<<<<<<< HEAD
```
Comment (wallinb) [inline amigos/amigos/Weather_Sensor_Data.py]: These merge conflict annotations can be cleaned up

File: Makefile [hunk 2]
```diff
@@ -1,28 +0,0 @@
-.PHONY: clean clean-pyc clean-build help
```
Comment (wallinb) [inline Makefile]: Don't delete this

File: README.md [hunk 3]
```diff
@@ -1,60 +0,0 @@
-# AMIGOS III operations command line interface
```
Comment (wallinb) [inline README.md]: Don't delete this

File: environment.yml [hunk 4]
```diff
@@ -1,49 +0,0 @@
-name: amigos
```
Comment (wallinb) [inline environment.yml]: Don't delete this

File: pytest.ini [hunk 5]
```diff
@@ -1,4 +0,0 @@
-[pytest]
```
Comment (wallinb) [inline pytest.ini]: Don't delete this

File: setup.cfg [hunk 6]
```diff
@@ -1,2 +0,0 @@
-[flake8]
```
Comment (wallinb) [inline setup.cfg]: Don't delete this

File: setup.py [hunk 7]
```diff
@@ -1,17 +0,0 @@
-#!/usr/bin/env python
```
Comment (wallinb) [inline setup.py]: Don't delete this

File: README.md (hunk 3 above)
Comment (CooviM) [reply README.md]: restor

File: pytest.ini (hunk 5 above)
Comment (CooviM) [reply pytest.ini]: restore

**Pull request closed without merging**